lib/ecto_schema_directories.ex

```diff
@@ -10,28 +10,31 @@ defmodule Nicene.EctoSchemaDirectories do
 
   @doc false
   def run(source_file, params \\ []) do
-    run(source_file, params, other_files(source_file.filename))
-  end
-
-  def run(source_file, params, other_files) do
-    issue_meta = IssueMeta.for(source_file, params)
-    is_schema = Credo.Code.prewalk(source_file, &schema?/2, false)
+    is_schema? = Credo.Code.prewalk(source_file, &schema?/2, false)
```

Comment:
turns out that `run/3` was used by credo for checks that should run on all files, and should not be defined anymore. We get a compilation warning on 1.6.0.

As we just want a public function that we can test, I've renamed it to `ensure_all_siblings_are_schema/3`